src/audiovideocontroller/AudioVideoControllerFacade.ts

```diff
@@ -27,6 +27,11 @@ export default interface AudioVideoControllerFacade {
    */
   start(options?: { signalingOnly?: boolean }): void;
   stop(): void;
+  /**
+   * This API will be deprecated in favor of `ClientMetricReport.getRTCStatsReport()`.
```

Comment:
Can we mention that  calling this will cause additional call to `getStats`?

Reply:
Added.